dev/benchmarks/complex_layout/android/gradle.properties

```diff
@@ -1,4 +1,3 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
-android.enableR8=true
```

Comment:
can you elaborate? Or is this just to save time since no one is shipping it?

Reply:
R8 has become the default in this AGP version. Having this causes a warning.

Reply:
got it, thanks